code/modules/tips/tips.dm

```diff
@@ -0,0 +1,233 @@
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/runechat
+	title = "Runechat"
+	desc = "In your options tab, 'runechat' refers to animations of the text said by players being displayed over their heads as they are talking."
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/attack_anims
+	title = "Attack animations"
+	desc = "Confusing about who's hitting who? You can toggle attack animations in your preferences."
+	category = "General"
+	weight = 3
+
+/datum/pomf_tip/vox_rights
+	title = "Vox rights!"
+	desc = "Vox rights are NOT human rights. Make sure to consult with the silicons and command staff before doing anything you'll regret when playing one!"
+	category = "General"
+	weight = 1
+
+// -- Command
+
+/datum/pomf_tip/chain_of_command
+	title = "Chain of command"
+	desc = "When there is no captain at roundstart, other heads of staff can become acting captain and secure important items such as the captain's spare and do job changes."
+	category = "Command"
+	weight = 3
+
+// -- Silicon laws
+
+/datum/pomf_tip/emotional_harm
+	title = "Emotional harm"
+	desc = "On the Asimov lawset, emotional harm is not considered human harm."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/pomf_tip/self_harm
+	title = "Self harm"
+	desc = "On the Asimov lawset, self-harm is not considered human harm."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/pomf_tip/burgs
+	title = "Robot names"
+	desc = "As a cyborg, minor misspelling of your name are NOT a good reason to ignore orders."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/pomf_tip/law_orders
+	title = "Self harm"
+	desc = "When they blatantly conflict, silicon laws are to be interpreted in descending order - higher laws have higher priority - unless otherwise specified. When obeying a higher law, you must avoid violating lower laws if possible."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/pomf_tip/slaved_borg
+	title = "Cyborgs are slaved to their AI"
+	desc = "As a cyborg slaved to an AI, you are expected (as far as reasonably possible) to obey your AI when it does not come into conflicts with your laws."
+	category = "Silicon laws"
+	weight = 3
+
+/datum/pomf_tip/mommis
+	title = "MoMMis"
+	desc = "Non-emagged MoMMis are not meant to interact with the crew too much - but a little wiggling has never hurt anybody."
+	category = "Silicon laws"
+	weight = 3
+
+// -- Science
+
+/datum/pomf_tip/xenoarch
+	title = "Artifacts"
+	desc = "Bringing an Artifact to the station is risky for your health! Reconsider, before people reconstruct your body into burgers."
+	category = "Science"
+	weight = 3
+
+/datum/pomf_tip/science
+	title = "Science goodies"
+	desc = "Science can do great things with what they have. Encourage them to be active, they usually just become hermits."
+	category = "Science"
+	weight = 3
+
+// -- Cargo
+
+/datum/pomf_tip/materials
+	title = "Materials"
+	desc = "Remember to give some materials to the Mechanics as a Miner, Science always hoards that stuff."
```

Comment:
a semicolon is more appropriate if stringing together two dependent clauses; alternatively you can include a conjunction like "because"